syntax_checkers/java/ensime.vim

```diff
@@ -0,0 +1,35 @@
+let s:save_cpo = &cpo
+set cpo&vim
+
+function! SyntaxCheckers_java_ensime_IsAvailable() dict
+    return 1
+endfunction
+
+function! SyntaxCheckers_java_ensime_GetLocList() dict
+    if exists('b:ensime_notes')
+        return b:ensime_notes
+    else
+        return []
+    endif
+endfunction
+
+function! SyntaxCheckers_java_ensime_GetHighlightRegex(error)
```

Comment:
Well, in order to make errors look just like it currently looks for scala, it is required (see SyntaxCheckers_scala_ensime_GetHighlightRegex). I could extract duplicated code in a separate function, but there a few lines. Should I still do it?